File: doc/palindrome.java
public class Palindrome{  
 public static void main(String args[]){  
  int r,sum=0,temp;    
  int n=454;// nombre à verifier
  
  temp=n;    
  while (n>0){    
   r=n%10;  
   sum=(sum*10)+r;    
   n=n/10;    
  }    
  if (temp==sum)    
   System.out.println (temp +" est un palindrome ");    
  else    
   System.out.println (temp + " n'est pas un palindrome");    
}  
}  

File: doc/palindrome.py

def palin(nbr):
	
    nbrInverser = 0
    tmp = nbr
    while (tmp != 0):
    	nbrInverser = nbrInverser * 10
    	nbrInverser = nbrInverser + tmp%10
    	tmp = tmp/10
       
     
    if (nbr == nbrInverser):
        print ("{} est un nombre palindrome.\n".format(nbr))
    else:
        print ("{} n'est pas un nombre palindrome.\n".format(nbr))
     
    return 0


####Programme principale
nbre=int (input ("Entrez un nombre pour verifier s'il s'agit d'un palindrome ou non\n"))
palin(nbre)
